.github/workflows/rails_cd.yml

```diff
@@ -7,7 +7,7 @@ jobs:
   continuous-deployment:
     runs-on: self-hosted
     steps:
-      - uses: tedconf/setup-ruby@v1.1.5
+      - uses: tedconf/setup-ruby@1.1.6
```

Comment:
GH seems to prefer using a leading `v` for version tags, cf. https://github.com/actions/toolkit/blob/main/docs/action-versioning.md, which is what I've started going (you can see when I made that decision), so in the interest of consistency, how about `v1.1.6`?